README.md

```diff
@@ -10,6 +10,12 @@ To install the collection from Ansible Galaxy:
 ansible-galaxy collection install crowdstrike.falcon
 ```
 
+## Installing on MacOS
+
+Due to Apple's decision to require Mobile Device Management (MDM) software to install kernel extensions without user prompting,
+the ansible collection with its roles will only be able to run on macOS in an interactive session which severely limits the
+power of Ansible.
```

Comment:
The snark! It burns!

Reply:
haha.... but seriously, snark was not intentional at all. Suggestions for change? Trying to let users know that the collection can be used to install the sensor, but only locally due to prompts because of the MDM requirement.